test/e2e/network_test.go

```diff
@@ -48,9 +48,8 @@ const (
 
 // testEastWestNetworking deploys Windows and Linux pods, and tests that the pods can communicate
 func testEastWestNetworking(t *testing.T) {
-	testCtx, err := NewTestContext(t)
+	testCtx, err := NewTestContext()
 	require.NoError(t, err)
-	defer testCtx.cleanup()
```

Comment:
General question: have you verified that we are deleting all the resources we are creating? This becomes more important when we want to run e2e tests multiple times locally.

Reply:
Ping on this

Reply:
I think we are, we also make sure to delete the namespace the resources are created in. That really should be moved out of the `delete` tests, and into the main `TestWMCO` function, as its going to cause issues rerunning the tests locally, as if the test attempts to create the namespace and it already exists its going to throw an error.

Reply:
I did have to add https://github.com/openshift/windows-machine-config-operator/pull/314/commits/746abd1de5fcd2a55a27cdf41516a9a7161a3382#diff-84675b994e150b2fe4128b3f0a171854d1060ba909b9ff0aeb8833c8d9f4dae9R163

Reply:
OK, just making sure we are not leaving stale resources around mainly for local testing like you mentioned.